tests/ttnn/unit_tests/gtests/tensor/test_sharding_with_alignment.cpp

```diff
@@ -258,15 +141,21 @@ TEST_P(ShardWithAlignmentTests, LogicalToPhysical) {
     const auto& logical_data = params.inputs.logical_data;
     const auto& expected_physical_data = params.expected.physical_data;
 
-    auto physical_data = convert_fp32_logical_data_to_physical_data(logical_data, tensor_spec);
+    // Convert output physical data to row major (if necessary) for testing
+    auto physical_data = tensor_impl::convert_logical_data_to_physical_data(logical_data, tensor_spec);
```

Comment:
Thinking of APIs. Wdyt about smth less verbose?
```
auto encoded_data = encode_tensor_data(logical_data, tensor_spec);
```

Reply:
I am okay to rename to whatever, but might not be worth it right now. We should probably align on terms like physical vs. encoded data (and logical vs. raw?) across the stack if you want to commonize to something less verbose.